build/scripts/check-toolset-insertion.ps1

```diff
@@ -0,0 +1,126 @@
+Param(
+    [string]$sourcePath = $null,
+    [string]$binariesPath = $null
+)
+set-strictmode -version 2.0
+$ErrorActionPreference="Stop"
+
+[Void][Reflection.Assembly]::LoadWithPartialName('System.IO.Compression.FileSystem')
```

Comment:
My understanding is that powershell will exit with `1` in case of exceptions?
I'll move it inside try/catch just in case. But experimenting proved the above.